components/infobox/commons/infobox_person.lua

```diff
@@ -144,7 +144,10 @@ function Person:createInfobox()
 				}}
 			end}
 		}},
-		Cell{name = 'Alternate IDs', content = {args.ids or args.alternateids}},
+		Cell{name = 'Alternate IDs', content = {
+				table.concat(mw.text.split(args.ids or args.alternateids or '', ',%s?'), ' &#8226; ')
```

Comment:
iirc performance wise doing `Array.map(mw.text.split(), function() return mw.text.trim() end)` is better

Reply:
Ah, so using a plain pattern without the optional space?

Reply:
yeah the trim removes the spaces